lib/classes/PluginManager.js

```diff
@@ -429,6 +430,29 @@ class PluginManager {
     this.getCommand(commandsArray);
   }
 
+  /**
+   * If the command has no use when operated in a working directory with no serverless
+   * configuration file, throw an error
+   */
+  validateServerlessConfigDependency(command) {
+    if (_.has(command, 'configDependent') && command.configDependent === true) {
+      return getServerlessConfigFile(process.cwd()).then((result) => {
```

Comment:
using cwd here rather than `sls.servicePath` is inconsistent (as I think you've noticed).  happy to adjust and fix it in #5314 but I'd be plenty happy with closing mine if it gets fixed here